meetings/2021-09-08.md

```diff
@@ -0,0 +1,140 @@
+
+**Henry** 
+ It is a great discussion. But it is branching a lot. Whould we move it over to the discussions area, which also has threading?
+
+ I am trying to find mathematical backing for the decisions we could make, so that we don't have to jump back and forward all the time. I think we have clearly Read/Write (those come from Lenses) and there could be subsets under Write for Create Update Delete. 
+Delete is an interesting question: does it need permissions on the container?
+
+**Justin** I don't forsee coming back 6 months later or 6 years later and saying - i need another mode.
+There has been some arguments that we will add some later, I think that our current set needs improvement
+but after that they should possibly last forever.
+
+**Barath** I agree, that well chosen modes will last for long time. Even substandard modes serve in existing systems.
+
+**Justin** I think delete is good example, it can be set on the level of container and on individual resources. I don't see argument that there could be delete without implicit update of containment triples.
+Argument that something else gets changed doesn't sound for me like recipe for good authorization system.
+
+**Pavlik** I think create and delete without allowing updating client managed triples in the
+container is a good scenario that is needed to be addressed.
+
+**Sarven** If you look at access modes in a vacum there may be all kind of modes that can be discussed.
+If we have protocol which has hierarchical containment ...
+Protocol usually starts with read and write. When you interacting - making a request against specific
+type of resource eg. container. It is processing operation, where protocol defines what happens.
+In WAC in order to delete something, one needs to have write on the resource and write on container
+containing it.
+
+Another thing that seems to be conflated is resource level and content level.
+
+[isssue 121: Towards an authorization framework for the Solid ecosystem](https://github.com/solid/authorization-panel/issues/121)
+We still haven't solved that issue, it can help us clarify where each use case fits in.
+I think we are ripping this issue apart instead of tackling it directly.
+I see operations in the protocol territory. I don't think solution needs to demand some particular
+access modes.
+
+If we look at PUT, from HTTP perspective you either create issue first or in subsequent requests
+you replace / update it.
+
+**Henry** I've been looking for some math that could help put some clarity in the problem. We are dealing with a prtocol between a client and a server, so that for example limits what we need to cover: we don't need to cover that the server has writes to make the changes.
+We sould assume that server has rights to do what it needs to do. I was mentioning Lenses: these are a generalisation of what we know in Java Beans with getters and setters. Having getters and setters one can do CRUD.
```

Comment:
```suggestion
**Henry** I've been looking for some math that could help put some clarity in the problem. We are dealing with a protocol between a client and a server, so that, for example, limits what we need to cover: we don't need to cover that the server has rights to make the changes.
We should assume that server has rights to do what it needs to do. I was mentioning Lenses: these are a generalisation of what we know in Java Beans with getters and setters. Having getters and setters, one can do CRUD.
```